File: BOJ/python3/4153.py
'''
직각삼각형
https://www.acmicpc.net/problem/4153
'''
import sys
import time
start = time.time()  # 시작 시간
sys.stdin = open("input.txt", "r")  # input.txt 파일로 입력


while(True):
    sides = list(map(int, input().split()))
    if sum(sides) == 0:
        break

    sides.sort()
    if sides[0]**2 + sides[1]**2 == sides[2]**2:
        print("right")
    else:
        print("wrong")


# 현재시각 - 시작시간 = 실행 시간(sec)
print("--------------------")
print(f"Elapsed Time: {round(time.time() - start, 3)}s")

File: BOJ/nodejs/4153.js
/*
직각삼각형
boj.kr/4153
 */
const start = new Date().getTime();

const fs = require("fs");
const input = fs.readFileSync("input.txt").toString().split("\n");
//const input = fs.readFileSync("/dev/stdin").toString().split("\n");

let line = 0;
while (input[line] !== "0 0 0") {
  const sides = input[line++].split(" ").map(Number);
  sides.sort((a, b) => a - b);
  console.log(
    sides[0] ** 2 + sides[1] ** 2 === sides[2] ** 2 ? "right" : "wrong"
  );
}

const end = new Date().getTime();
console.log("--------------------");
console.log(`Elapsed Time: ${(new Date().getTime() - start) / 1000}s`);
